Makefile

```diff
@@ -73,7 +73,7 @@ shell: ## Exec into a container with the kubicorn source mounted inside
 	--rm ${SHELL_IMAGE} /bin/bash
 
 lint: install-tools ## check for style mistakes all Go files using golint
-	golint $(PKGS)
+	~/go/bin/golint $(PKGS)
```

Comment:
We shouldn't do this, not everyone has go tools in this location, general approach is to add $GOPATH/bin to $PATH env variable. So, that golint or goimports commands can work. This commands will for sure fail on my host.